build/build.fsx

```diff
@@ -1,9 +1,17 @@
 #r "paket:
+frameworks: net6.0
+source https://api.nuget.org/v3/index.json
```

Comment:
source is needed, fails otherwise. I don't know why. The error:

```
Could not load types of compiled script:
 - Method 'Equals' in type 'BuildVersion' from assembly 'build_5FA8DE18770076A3BEEA4F5E75F33168E0469023020F76451FB607F0662FFED7, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
 - Method 'Equals' in type 'LiquidTag' from assembly 'build_5FA8DE18770076A3BEEA4F5E75F33168E0469023020F76451FB607F0662FFED7, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
 - Method 'Equals' in type 'Declaration' from assembly 'build_5FA8DE18770076A3BEEA4F5E75F33168E0469023020F76451FB607F0662FFED7, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
 - Method 'Equals' in type 'Snippet' from assembly 'build_5FA8DE18770076A3BEEA4F5E75F33168E0469023020F76451FB607F0662FFED7, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
 - Method 'Equals' in type 'LiquidTag' from assembly 'build_5FA8DE18770076A3BEEA4F5E75F33168E0469023020F76451FB607F0662FFED7, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
 - Method 'Equals' in type 'LiquidTag' from assembly 'build_5FA8DE18770076A3BEEA4F5E75F33168E0469023020F76451FB607F0662FFED7, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
 - Method 'Equals' in type 'LiquidTag' from assembly 'build_5FA8DE18770076A3BEEA4F5E75F33168E0469023020F76451FB607F0662FFED7, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
Script reported an error:
-> MissingMethodException: Method not found: 'System.String Fake.Core.Environment.environVarOrDefault(System.String, System.String)'.

```